sql/updates/world/3.3.5/9999_99_99_99_world.sql

```diff
@@ -0,0 +1,95 @@
+(0, 36460, 64, '', '', 'Disable LOS - Ultra Deconsolodation Zapper'),
+(0, 45323, 64, '', '', 'Disable LOS - Returning Vrykul Artifact'),
+(0, 45425, 64, '', '', 'Disable LOS - Shoot (Dummy)'),
+(0, 45537, 64, '', '', 'Disable LOS - Lightning Beam Channel'),
+(0, 45724, 64, '', '', 'Disable LOS - Braziers Hit'),
+(0, 45949, 64, '', '', 'Disable LOS - Release Aberration'),
+(0, 46171, 64, '', '', 'Disable LOS - Scuttle Wrecked Flying Machine'),
+(0, 46314, 64, '', '', 'Disable LOS - Slippery Floor Ambient'),
+(0, 46422, 64, '', '', 'Disable LOS - Shamans Look for Opening'),
+(0, 46593, 64, '', '', 'Disable LOS - Whisp Flight Missile and Beam'),
+(0, 46603, 64, '', '', 'Disable LOS - Whisp to Flight'),
+(0, 46836, 64, '', '', 'Disable LOS - Spell Flame Patch'),
+(0, 46842, 64, '', '', 'Disable LOS - Flame Ring'),
+(0, 47117, 64, '', '', 'Disable LOS - Summon Image of Drakuru 01'),
+(0, 47149, 64, '', '', 'Disable LOS - Summon Image of Drakuru 02'),	
+(0, 47316, 64, '', '', 'Disable LOS - Summon Image of Drakuru 03'),
+(0, 47405, 64, '', '', 'Disable LOS - Summon Image of Drakuru 04'),
+(0, 48188, 64, '', '', 'Disable LOS - Flask of Blight'),
+(0, 49308, 64, '', '', 'Disable LOS - Utgard Pinacle Reset Check'),
+(0, 50313, 64, '', '', 'Disable LOS - Mole Machine Emerge'),
+(0, 50439, 64, '', '', 'Disable LOS - Summon Image of Drakuru 05'),
+(0, 50563, 64, '', '', 'Disable LOS - Carve Stone'),
+(0, 51964, 64, '', '', 'Disable LOS - Tormentor\'s Incense'),
+(0, 52212, 64, '', '', 'Disable LOS - Death and Decay'),
+(0, 52227, 64, '', '', 'Disable LOS - Dilute Blight Cauldron'),
+(0, 52228, 64, '', '', 'Disable LOS - Kill Credit (quest 12669)'),
+(0, 53038, 64, '', '', 'Disable LOS - Didgeridoo of Contemplation'),
+(0, 54114, 64, '', '', 'Disable LOS - Heart of the Phoenix'),
+(0, 56940, 64, '', '', 'Disable LOS - Thorim Story Kill Credit'),
+(0, 57090, 64, '', '', 'Disable LOS - Wyrmrest Skytalon - Revivify'),
+(0, 56091, 64, '', '', 'Disable LOS - Wyrmrest Skytalon - Flame Spike'),
+(0, 56092, 64, '', '', 'Disable LOS - Wyrmrest Skytalon - Engulf in Flames'),
+(0, 57143, 64, '', '', 'Disable LOS - Wyrmrest Skytalon - Life Burst'),
+(0, 57431, 64, '', '', 'Disable LOS - Summon Static Field'),
+(0, 57473, 64, '', '', 'Disable LOS - Arcane Storm'),
+(0, 58515, 64, '', '', 'Disable LOS - Burn Corpse'),
+(0, 58518, 64, '', '', 'Disable LOS - Ahn kahet Brazier Kill Credit'),
+(0, 62042, 64, '', '', 'Disable LOS - Stormhammer'),
+(0, 62266, 64, '', '', 'Disable LOS - Trigger 3 Adds'),
+(0, 64767, 64, '', '', 'Disable LOS - Stormhammer'),
+(0, 66636, 64, '', '', 'Disable LOS - Rising Anger'),
+(0, 69922, 64, '', '', 'Disable LOS - Temper Quel Delar'),
+(0, 69956, 64, '', '', 'Disable LOS - Return Tempered Quel Delar'),
+(0, 70460, 64, '', '', 'Disable LOS - Coldflame Jets'),
+(0, 71024, 64, '', '', 'Disable LOS - Throw Bomb'),
+(0, 71599, 64, '', '', 'Disable LOS - Explosion (Chemical Wagon)'),
+(0, 75509, 64, '', '', 'Disable LOS - Twilight Mending');
```

Comment:
missing new line

Reply:
(Fixed now, but github can't hide comments after last line in the files)